pkg/render/kube-controllers.go

```diff
@@ -134,7 +134,7 @@ func (c *kubeControllersComponent) Objects() ([]client.Object, []client.Object)
 			secret.CopyToNamespace(common.CalicoNamespace, c.elasticsearchSecret)...)...)
 	}
 
-	if c.managementCluster != nil && c.elasticsearchUserSecret != nil {
+	if !c.isManagedCluster() && c.elasticsearchUserSecret != nil {
```

Comment:
If the contents of the secrets change, is there an annotation to make sure the pod restarts?

Reply:
@rene-dekker Yup, we should have watched added for them.

Reply:
In addition to the watch we also annotate the podTemplate with a hash for this secret so that ensures it will be restarted in the case of change.